sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -175,13 +175,13 @@ class HttpTransport(
 
     @abc.abstractmethod
     def send(self, request, **kwargs):
-        # type: (PipelineRequest, Any) -> PipelineResponse
+        # type: (PipelineRequest, Any) -> HTTPResponseType
         """Send the request using this HTTP sender.
 
         :param request: The pipeline request object
         :type request: ~azure.core.pipeline.PipelineRequest
         :return: The pipeline response object.
-        :rtype: ~azure.core.pipeline.PipelineResponse
+        :rtype: ~azure.core.pipeline.transport.HttpResponse
```

Comment:
Are AsyncHttpTransport typing/docstring correct?

Reply:
It does not have docstring. I added.

Reply:
We don't have typing/docstring for AsyncHttpTransport. So it is not incorrect. :)

Reply:
Might be the opportunity to add them, but yeah optional :p